tests/end-to-end/external-solution-integration/internal/scenario/event_mesh_prepare/event_mesh_prepare.go

```diff
@@ -0,0 +1,93 @@
+package event_mesh_prepare
+
+import (
+	serviceCatalogClient "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+	log "github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	coreClient "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	eventing "knative.dev/eventing/pkg/client/clientset/versioned"
+
+	appBrokerClient "github.com/kyma-project/kyma/components/application-broker/pkg/client/clientset/versioned"
```

Comment:
imports should not be in camelcases. Please use: `appbrokerclient` `coreclient` etc.